go/serial/stat.fbs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+include "prolly.fbs";
+
+namespace serial;
+
+// StoreRoot is the top-level chunk in the store,
+// containing references to all named Refs.
+table Statistic {
```

Comment:
Is the plan to add more fields to this over time? Otherwise I'm not sure why the root doesn't just references the table address directly instead of going through this.

Unless I'm misunderstanding the purpose of this message type.

Reply:
Aaron and I talked about how it might be better to have a dedicated message type for this, otherwise the helper functions that limit the acceptable root types would have to include the generic prolly node message type.